SEOImages: User Story 1463880, Part 36 (#11451)
dotnet/docs: docs/framework/wcf/feature-details/wcf-web-http-service-help-page.md

# WCF Web HTTP Service Help Page
[!INCLUDE[netfx_current_long](../../../../includes/netfx-current-long-md.md)] provides an automatic help page for WCF WEB HTTP services. This help page lists a description of each operation, request and response formats, and schemas. This functionality is turned off by default. When a user browses to a WCF WEB HTTP service and appends "/Help" on to the end of the URL, for example `http://localhost:8000/Customers/Help`, a help page like the following is displayed.  
  

>  ![Screenshot that shows the WCF REST Help page.](./media/wcf-web-http-service-help-page/windows-communication-foundation-rest-help-page.gif)  

Reviewer: ```suggestion
 ![A browser with the WCF REST Help page open.](./media/wcf-web-http-service-help-page/windows-communication-foundation-rest-help-page.gif)  
```

  
 The user can then click any method listed in the help page and detailed page for that operation is displayed showing more information about the method, including message formats and example responses. The following image is an example of a help page for a method.  
  

>  ![Screenshot showing WCF REST Help Page details.](./media/wcf-web-http-service-help-page/windows-communication-foundation-rest-help-page-detail.gif)  

Reviewer: ```suggestion
 ![A browser with the WCF REST Help page details for the GetCustomers method open.](./media/wcf-web-http-service-help-page/windows-communication-foundation-rest-help-page-detail.gif)  
```
